packages/dates/src/date-time-utils/DateTimeUtils.ts

```diff
@@ -146,41 +145,93 @@ export function formatDate(
   return date.toLocaleString(DATE_DISPLAY_VARIANTS[variant]);
 }
 
+function formatUnit(unit: ToRelativeUnit): string {
+  switch (unit) {
+    case 'quarters':
+      return 'q';
```

Comment:
why don't you simplify it?
```suggestion
      return unit.charCodeAt(0);
```